database_manager/src/run_server.cpp

```diff
@@ -19,6 +20,10 @@
 #include "toaster_msgs/Fact.h"
 #include "toaster_msgs/Id.h"
 #include "toaster_msgs/FactList.h"
+#include <fstream>
```

Comment:
Please chack that those 3 includes are needed. They are probably already included in other included files.